components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/constant/ActionMgtConstants.java

```diff
@@ -53,19 +55,30 @@ public enum ErrorMessages {
                 "%s is empty."),
         ERROR_INVALID_ACTION_REQUEST_FIELD("60005", "Invalid request.",
                 "%s is invalid."),
+        ERROR_INVALID_ACTION_CERTIFICATE("60006", "Invalid request.", "Provided certificate is invalid."),
```

Comment:
This is specific to a particular extension, we should be able to take these validations out from the core

Reply:
Fixed